lib/routes/tongji/sem/_utils.ts

```diff
@@ -0,0 +1,71 @@
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+import { config } from '@/config';
+
+export async function getNotifByPage(pageNumber: number) {
+    const pageUrl: string = `https://sem.tongji.edu.cn/semch/category/frontpage/notice/page/${pageNumber}`;
+
+    try {
+        const response = await got.get(pageUrl, {
```

Comment:
Use the [redirected URL](https://github.com/DIYgod/RSSHub/actions/runs/11757146978/job/32754065160#step:13:21) instead.